woocommerce/woocommerce-android
# [Woo POS] Implement the UI of the product list

Closes: #11785 


### Description

* Adds the UI of the products list
* Adds reusable shimmer component 
* Adds loading state and "loading more" indicator
* Base for adding custom colors to the theme

We will still need to implement these states:
* Empty
* Error
* Add pull to refresh

### Testing information

* Point attention to the UI
* Try opening it when there are no products in the cache to validate
* Load more and notice loading more indicator

### Images/gif



https://github.com/woocommerce/woocommerce-android/assets/4923871/77174a67-60a8-4a02-8b5f-9cdeaec610cf
